corgi/tasks/prod_defs.py

```diff
@@ -323,22 +323,7 @@ def _parse_cpes_from_brew_tags(
     version_name: str,
 ) -> list[str]:
     """Match streams using brew_tags to Errata Tool Product Versions and their Variants"""
```

Comment:
Maybe a small update to the comment is helpful:
```suggestion
    """Match streams using brew_tags to Errata Tool Product Versions and their Variants
       Return a list of CPEs for the matched Variants"""
```

Reply:
updated